spec/intl.html

```diff
@@ -1338,6 +1338,101 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateerayear">
+          <h1>CalendarDateEraYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateEraYear performs implementation-defined processing to find the year of the current era for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateyear">
+          <h1>CalendarDateYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateYear performs implementation-defined processing to find the year for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonth">
+          <h1>CalendarDateMonth ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateMonth performs implementation-defined processing to find the month for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonthcode">
+          <h1>CalendarDateMonthCode ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateMonthCode performs implementation-defined processing to find the monthCode corresponding to the month for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateday">
+          <h1>CalendarDateDay ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDay performs implementation-defined processing to find the day for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedayofweek">
+          <h1>CalendarDateDayOfWeek ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDayOfWeek takes a date-like object _date_ and returns the day of the week in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedayofyear">
+          <h1>CalendarDateDayOfYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDayOfYear takes a date-like object _date_ and returns the day of the year in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateweekofyear">
+          <h1>CalendarDateWeekOfYear ( _calendar_, _date_ )</h1>
```

Comment:
Should there be discussion of how this prop differs from ISO?  (I'm admittedly not sure how it does, but seems worth discussing)